packages/react-examples/src/react/Icon/Icon.ImageSheet.Example.tsx

```diff
@@ -41,31 +41,28 @@ const classNames = mergeStyleSets({
 export const IconImageSheetExample: React.FunctionComponent = () => {
   // ImageIcon is an optimized variant of standard Icon.
   // You could also use the standard Icon here (adding the prop `iconType={IconType.image}`).
-  // Provide an `aria-label` (and set `role="img"`) for screen reader users if the icon is not accompanied by text
-  // that conveys the same meaning.
+  // Provide an alt attribute using imageProps.alt for screen reader users if the icon is not
+  // accompanied by text that conveys the same meaning.
   return (
     <div>
       <ImageIcon
-        aria-label="OneNote file icon"
-        role="img"
         className={classNames.one}
         imageProps={{
           src: TestImages.iconOne,
+          alt: 'OneNote file',
           className: css(classNames.image, classNames.oneImage),
         }}
       />
       <ImageIcon
-        aria-label="Check icon"
-        role="img"
         className={classNames.check}
         imageProps={{
           src: TestImages.iconOne,
+          'aria-label': 'Printer ready',
```

Comment:
Why are you using `alt` above and `aria-label` here? It seems like they are effectively the same.

Reply:
It was partly to demo different options for accessible naming, partly so there were snapshot tests for both :D.

I have no idea if examples is the right place for either of those though, so happy to change them to all use `alt`, which is more of a best practice :D

Reply:
Having snapshot tests (and/or tests explicitly verifying that attributes are applied) for the different accessible name scenarios would actually be a great idea. You can follow the patterns of the other snapshot tests in Icon.test.tsx, FontIcon.test.tsx, and we should probably add ImageIcon.test.tsx.

Reply:
updated! I added tests, and changed the example to use just `alt`